sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPSensor.java

```diff
@@ -77,17 +78,19 @@
   private final FilePredicate mainFilePredicate;
   private final FileLinesContextFactory fileLinesContextFactory;
   private final Checks<CodeVisitor> checks;
+  private final NoSonarFilter noSonarFilter;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "noSonarFilter" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.internal.sonarsource.com/coding_rules#rule_key=squid%3AS1068)